platform/centec-arm64/tsingma-bsp/src/ctcmac/ctcmac.c

```diff
@@ -1736,7 +1736,8 @@ static bool ctcmac_new_page(struct ctcmac_priv_rx_q *rxq,
 	struct page *page;
 	dma_addr_t addr;
 
-	page = dev_alloc_page();
+	//ipage = dev_alloc_page();
+	page = __dev_alloc_pages(GFP_DMA | GFP_ATOMIC | __GFP_NOWARN, 0);
```

Comment:
this file is owned by centec, can you check centrc folks to sign off this change? this also looks like a bug fix, so it would be better if we can have this a separate pr and have centec folks to sign off.

Reply:
Revert this revision, which originated from a bug in the management port eth0 of our product and Centec's adaptation.  It will not affect the compilation of this PR.